server/src/main.rs

```diff
@@ -9,6 +11,17 @@ async fn main() -> Result<(), error::Error> {
 
     info!("Starting Stackable Operator for Apache Kafka");
     let client = client::create_client(Some(FIELD_MANAGER.to_string())).await?;
+
+    if let Err(error) = stackable_operator::crd::wait_until_crds_present(
+        &client,
+        vec![&KafkaCluster::crd_name()],
+        None,
+    )
+    .await
+    {
+        error!("Required CRDs missing, aborting: {:?}", error);
```

Comment:
This doesn't seem to actually abort, it just says it'll abort and then starts spamming the log because it still starts the server..
It shouldn't return as no timeout is given, but if we later decide to add a timeout this could come back to bite us.